boot/zephyr/main.c

```diff
@@ -216,7 +299,7 @@ void main(void)
 #else
                             GPIO_DIR_IN | GPIO_PUD_PULL_UP
 #endif
-	    );
+           );
```

Comment:
Unnecessary whitespace change

Reply:
I replaced Tab indentation. spaces indentation is mcuboot CS contention,